src/errorHandler/errorHandler.spec.ts

```diff
@@ -169,25 +186,31 @@ describe('Server error handling: ', () => {
         }
       }
     `;
+    const { body } = await server.executeOperation({ query });
+    assert(body.kind === 'single');
+    const res = body.singleResult;
+    assert(res.errors !== undefined);
 
-    const res = await server.executeOperation({ query });
     expect(res.errors.length).to.equal(1);
     expect(res.errors[0].message).to.contain('Syntax Error');
     [consoleSpy, sentrySpy].forEach((spy) => {
       expect(spy.callCount).to.equal(0);
     });
   });
-  it('does not mask errors from apollo-server-errors raised by application', async () => {
+  it('does not mask GraphQLErrors with a defined extension', async () => {
```

Comment:
This is as close as I could figure to match the original intent.  Does this seem right?

Reply:
ya i think so..looking at the test, it tries to see if we get the graphQL error message and code set for the apollo errors